apps/design/backend/src/store.ts

```diff
@@ -213,6 +219,42 @@ export function generateBallotStyles(
   }
 }
 
+export function assignCustomContentToBallotStyles(
```

Comment:
This name sounds like it mutates something. Maybe call it `getNhCustomContentByBallotStyle`?